controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -5569,6 +6336,8 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           limit: req.limit,
           offset: req.offset,
           namespaceId: namespace?.id,
+          excludeMonographs: req.monographConfig.case === 'excludeMonographs' && req.monographConfig.value,
```

Comment:
Also here. Now a getFederatedGraphs method has to deal with monographs too. We all know why but this vague line is hard to reason about. Can we rename `getFederatedGraphs` to `getGraphs`? Alternatively, introduce new RPC methods to deliver only monographs and federated graphs.